src/fiatconnect/index.ts

```diff
@@ -53,11 +53,10 @@ export async function getFiatConnectProviders(
     `${networkConfig.getFiatConnectProvidersUrl}?address=${address}&providers=${providerList}`
   )
   if (!response.ok) {
-    Logger.error(
-      TAG,
-      `Failure response fetching FiatConnect providers: ${response} , returning empty list`
+    Logger.error(TAG, `Failure response fetching FiatConnect providers: ${response}`)
+    throw new Error(
+      `Failure response fetching FiatConnect providers. ${response.status}  ${response.statusText}`
     )
-    return []
```

Comment:
switched this to throw an error so that we use cached values of the provider list if this call fails